app/src/main/java/com/taylorgirard/comicconvo/activities/LoginActivity.java

```diff
@@ -53,8 +53,8 @@ public void onClick(View v) {
         btnSignUp.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                Intent i = new Intent(LoginActivity.this, SignUpActivity.class);
-                startActivity(i);
```

Comment:
test comment 2: Let's name intents as `intent` going forward. Only use `i` for iterators